storeavalue.php

```diff
@@ -1,14 +1,18 @@
 <?php
 
-$file = "database.txt";
+header("Content-Type: application/json");
+$file = "database.json";
+
+if (file_exists("database.txt") && !file_exists($file)) {
+    rename("data.txt", $file);
```

Comment:
I think this is wrong too

```suggestion
    rename("database.json", $file);
```